pom.xml

```diff
@@ -657,9 +662,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.0</version>
                 <configuration>
-                   <!-- testsToExclude come from the profile-->
-                   <excludedGroups>${testsToExclude}</excludedGroups>
-                    <argLine>${argLine} -Duser.timezone=${project.timezone} -Dfile.encoding=${project.build.sourceEncoding} -Duser.language=${project.language} -Duser.region=${project.region}</argLine>
+                    <!-- testsToExclude come from the profile-->
+                    <excludedGroups>${testsToExclude}</excludedGroups>
```

Comment:
Thanks for fixing the indentation here.

Reply:
I was in the area, you know... :smile:

Reply:
I have this dream that bit by bit the code will conform to our house style if we all fix up the lines we're touching. This is probably naive of me. 😄